langlib/lang.string/src/main/java/org/ballerinalang/langlib/string/EqualsIgnoreCaseAscii.java

```diff
@@ -59,13 +56,14 @@ public static boolean equalsIgnoreCaseAscii(BString s1, BString s2) {
     }
 
     private static boolean isPureAscii(String  str) {
```

Comment:
Shall we check whether we can replace this method's entire method by the following line?
`return StandardCharsets.US_ASCII.newEncoder().canEncode(str);`